File: Lógica de Programação Java e C#/4 - Controle de Fluxo/Java/Exercício de Fixação/EscolheRoupa.java
class EscolheRoupa{
	public static void main(String [] args){
		double valor = Math.random();
	if(valor<0.5){
	System.out.println("Camiseta Preta");
	}else{
	System.out.println("Camiseta vermelha");
	}
	valor=Math.random();
	if(valor<0.5){
	System.out.println("Calça Jeans");
	}else{
	System.out.println("bermuda");
	}
	valor=Math.random();
	if (valor<0.5){
	System.out.println("Tênis");
	}else{
	System.out.println("Sapato");
	}
	valor= Math.random();
	if (valor <0.5){
	System.out.println("Boné");
	}else{
	System.out.println("óculos");
	}
}
}

File: Lógica de Programação Java e C#/4 - Controle de Fluxo/C#/Exercícios de Fixação/EscolheRoupa.cs
class EscolheRoupa
{
    static void Main()
    {
        System.Random gerador = new System.Random();
        double valor = gerador.NextDouble();
        if (valor < 0.5)
        {
            System.Console.WriteLine("Camiseta Preta");
        }
        else
        {
            System.Console.WriteLine("Camiseta Vermelha");
        }
        valor = gerador.NextDouble();
        if (valor < 0.5)
        {
            System.Console.WriteLine("Calça jeans");
        }
        else
        {
            System.Console.WriteLine("Bermuda");
        }
        if (valor < 0.5)
        {
            System.Console.WriteLine("Tênis");
        }
        else
        {
            System.Console.WriteLine("Sapato");
        }
        if (valor < 0.5)
        {
            System.Console.WriteLine("Boné");
        }
        else
        {
            System.Console.WriteLine("óculos");
        }
    }
}